lib/features/photo_detail/photo_detail_widget_model.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:elementary/elementary.dart';
+import 'package:elementary_helper/elementary_helper.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:photo_stock/domain/photo/photo.dart';
+import 'package:photo_stock/features/photo_detail/photo_detail.dart';
+import 'package:photo_stock/features/photo_detail/photo_detail_model.dart';
+import 'package:photo_stock/util/app_dictionary.dart';
+import 'package:provider/provider.dart';
+
+/// Factory for [PhotoDetailWidgetModel]
+PhotoDetailWidgetModel photoDetailWMFactory(
+  BuildContext context,
+) {
+  final model = context.read<PhotoDetailModel>();
+
+  return PhotoDetailWidgetModel(model);
+}
+
+/// Widget Model for [PhotoDetailScreen]
+class PhotoDetailWidgetModel
+    extends WidgetModel<PhotoDetailScreen, PhotoDetailModel>
+    implements IPhotoDetailWidgetModel {
+  final _photoDetailState = EntityStateNotifier<Photo>();
+
+  @override
+  ValueListenable<EntityState<Photo>> get photoDetailState => _photoDetailState;
```

Comment:
Тк ты используешь интерфейс, то ты можешь вместо геттера и приватного поля объявить одно публичное поле (вот так):
```suggestion
  @override
  final EntityStateNotifier<Photo> photoDetailState = EntityStateNotifier<Photo>();
```

Reply:
Эту штуку пофиксил